Client/Frontend/Browser/BrowserViewController/BrowserViewController+Callout.swift

```diff
@@ -0,0 +1,159 @@
+            Preferences.General.basicOnboardingCompleted.value != OnboardingState.completed.rawValue
+
+        let showedPopup = Preferences.VPN.popupShowed
+
+        if onboardingNotCompleted
+            || showedPopup.value
+            || !VPNProductInfo.isComplete {
+            return
+        }
+        
+        let popup = EnableVPNPopupViewController().then {
+            $0.isModalInPresentation = true
+            $0.modalPresentationStyle = .overFullScreen
+        }
+        
+        popup.enableVPNTapped = { [weak self] in
+            self?.presentCorrespondingVPNViewController()
+        }
+        
+        present(popup, animated: false)
+        isfullScreenCalloutPresented = true
+        showedPopup.value = true
+    }
+    
+    func presentDefaultBrowserScreenCallout() {
+        if Preferences.DebugFlag.skipNTPCallouts == true, isfullScreenCalloutPresented { return }
+
+        if !FullScreenCalloutManager.shouldShowDefaultBrowserCallout(calloutType: .defaultBrowser) {
+            return
+        }
+                
+        let onboardingController = WelcomeViewController(
+            profile: nil,
+            rewards: nil,
+            state: WelcomeViewCalloutState.defaultBrowserWarning(
+                title: Strings.Callout.defaultBrowserCalloutTitle,
+                details: Strings.Callout.defaultBrowserCalloutDescription,
+                primaryButtonTitle: Strings.Callout.defaultBrowserCalloutPrimaryButtonTitle,
+                secondaryButtonTitle: Strings.Callout.defaultBrowserCalloutSecondaryButtonTitle,
+                primaryAction: {
+                    guard let settingsUrl = URL(string: UIApplication.openSettingsURLString) else {
+                        return
+                    }
+                    
+                    Preferences.General.defaultBrowserCalloutDismissed.value = true
+                    UIApplication.shared.open(settingsUrl)
+                }, secondaryAction: { [weak self] in
+                    self?.dismiss(animated: false)
+                })
+        )
+
+        present(onboardingController, animated: true)
+        isfullScreenCalloutPresented = true
+    }
+    
+    func presentBraveRewardsScreenCallout() {
+        if Preferences.DebugFlag.skipNTPCallouts == true, isfullScreenCalloutPresented { return }
+
+        if !FullScreenCalloutManager.shouldShowDefaultBrowserCallout(calloutType: .rewards) {
+            return
+        }
+                
+        if BraveRewards.isAvailable, !Preferences.Rewards.rewardsToggledOnce.value {
+            let controller = OnboardingRewardsAgreementViewController(profile: profile, rewards: rewards)
+            controller.onOnboardingStateChanged = { [weak self] controller, state in
+                self?.dismissOnboarding(controller, state: state)
+            }
+            present(controller, animated: true)
+            isfullScreenCalloutPresented = true
+
+        }
+        return
```

Comment:
this return can be removed

Reply:
Removed at https://github.com/brave/brave-ios/pull/4508/commits/429df1c4afce998ccc565b27120879a5afdfe87f